packages/builder/src/internal/cli/arguments-parser.ts

```diff
@@ -40,8 +42,19 @@ export class ArgumentsParser {
     );
   }
 
+  private _substituteShortParam(arg: string, algobShortParamSubs: AlgobShortParamSubstitutions): string {
```

Comment:
Why do you use `algob` prefix? It can be use with anything, not only algob. Le's remove it.

Reply:
I kept what I saw in 
`algobParamDefs: AlgobParamDefinitions,`
Will remove from that one too.